i18n/transifex.py

```diff
@@ -127,21 +129,18 @@ def clean_file(configuration, filename):
     Strips out the warning from a translated po file about being an English source file.
     Replaces warning with a note about coming from Transifex.
     """
-    try:
-        pofile = polib.pofile(filename)
-    except Exception as exc:  # pylint: disable=broad-except
-        # An exception can occur when a language is deleted from Transifex.
-        # Don't totally fail here.
-        print(
-            "Encountered error {} with filename {} - language project may "
-            "no longer exist on Transifex".format(exc, filename)
-        )
-        return
+    pofile = polib.pofile(filename)
+
+    # For 0% translated files, this metadata key will be missing, but our validate code requires it.
+    # Just make sure it is defined at all.
+    pofile.metadata.setdefault('Last-Translator', '')
```

Comment:
We will also have to set the 'Project-Id-Version', 'Language-Team', 'PO-Revision-Date'. if those are missing/default it will also cause the validate to fail i had to set those too in different repos when using i18n validate to validate translations before merge.

Reply:
Ah, I haven't noticed any problems with those headers. In a quick test, all those headers are set for empty po files from Transifex. Do you remember where those were causing problems?

Reply:
oh yeah you are right if they are set in source language files they will be set in untranslated po files too. i got confused with the issues i faced early when whole header was messed up.